src/Printer.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+const fileRelativePath = "../node_modules"
```

Comment:
/ symbol can be different in different OS systems and can be reason of bugs
I would suggest to change this code like 
`const filePath = path.resolve(__dirname, '..', 'node_modules)`